src/commands/doctor.ts

```diff
@@ -110,6 +125,7 @@ module.exports = {
       ...pnpmPackages,
       bunInfo,
       expoInfo,
+      ...(depExpoCliInfo ? [depExpoCliInfo] : []),
```

Comment:
just `depExpoCliInfo` is good here, no need to spread or send an empty array

Reply:
Gotcha ok, so just make the column3 adjustment for yellow correction and we'll call it good!